cc/transport/grpc_streaming_transport.h

```diff
@@ -96,18 +96,16 @@ class GrpcStreamingTransport : public TransportWrapper {
   absl::StatusOr<::oak::session::v1::ResponseWrapper> Send(
       const ::oak::session::v1::RequestWrapper& request) {
     // Send a request.
-    channel_reader_writer_->Write(request);
-    channel_reader_writer_->WritesDone();
+    if (!channel_reader_writer_->Write(request)) {
+      return absl::InternalError("couldn't send request");
+    }
 
     // Receive a response.
     ::oak::session::v1::ResponseWrapper response;
-    channel_reader_writer_->Read(&response);
-    ::grpc::Status status = channel_reader_writer_->Finish();
-    if (status.ok()) {
-      return response;
-    } else {
-      return absl::InternalError("couldn't send request: " + status.error_message());
+    if (!channel_reader_writer_->Read(&response)) {
+      return absl::InternalError("couldn't receive response");
     }
+    return response;
   }
 };
```

Comment:
Please add a deconstructor to close the stream.

Reply:
Done